test/integration/tas/tas_test.go

```diff
@@ -80,6 +80,58 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 			tasFlavor := testing.MakeResourceFlavor("tas-flavor").TopologyName("default").Obj()
 			gomega.Expect(k8sClient.Create(ctx, tasFlavor)).Should(gomega.HaveOccurred())
 		})
+
+		ginkgo.It("should not allow to update ResourceFlavorSpec", func() {
```

Comment:
I would propose to create separate describe for validation. Like we have for topology on L1040.

Reply:
Done